nano/node/election.cpp

```diff
@@ -50,11 +50,7 @@ void nano::election::confirm_once (nano::election_status_type type_a)
 		{
 			--node.active.long_unconfirmed_size;
 		}
-		auto root (status.winner->qualified_root ());
-		node.active.add_confirmed (status, root);
-		clear_blocks ();
```

Comment:
Not sure why clear_blocks (); & clear_dependent (); were removed here. Because after removing root we ideally should immediately clean blocks (for correct difficulty adjustment) & adjust difficulty

Reply:
I needed the blocks at a later time to get the election information, but am just making a new collection with the necessary ones now.